api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -114,6 +144,10 @@ message RBAC {
   // Maps from policy name to policy. A match occurs when at least one policy matches the request.
   // The policies are evaluated in lexicographic order of the policy name.
   map<string, Policy> policies = 2;
+
+  // Audit options that includes the condition for audit logging to happen and
+  // audit logger configurations.
+  AuditOptions audit_options = 3;
```

Comment:
I think this needs a `[#not-implemented-hide:]` comment.

Reply:
Thanks for the catch! Done.